src/components/NotificationBanner/NotificationBanner.stories.tsx

```diff
@@ -29,9 +29,6 @@ import {
 export default {
   title: 'Components/Notification/NotificationBanner',
   component: NotificationBanner,
-  parameters: {
-    layout: 'fullscreen',
-  },
```

Comment:
This was making the story less helpful because it was hiding the NotificationBanner styles:

<img width="1680" alt="Screenshot 2020-09-22 at 11 51 56" src="https://user-images.githubusercontent.com/35560568/93867967-0ecb0180-fcca-11ea-99a1-95565a5faf8b.png">